samples-logs-back-end/cronjob/sync.py

```diff
@@ -40,38 +45,56 @@ def collect_table_data():
     return results
 
 
-def create_tmp_collection(records):
-    """
-    This function will create new collection
-    :param records: list of records from data portal
-    """
-    for record in records:
-        DB.phylo_tmp.insert_one(record)
-
-
-def update_tmp_collection():
+def update_tmp_phylo_collection():
     """
     This function will add information about phylogenetic tree to collection
     and copy new collections to prod
     """
+    subprocess.run(
+        "wget http://45.86.170.46/coronavirus_sequence.tsv",
+        shell=True,
+        capture_output=True)
```

Comment:
Before downloading this file please add a process of a removal of an old file (otherwise you'll get coronavirus_sequence.tsv1 and will always read old file)

Reply:
Added `backups=1` to the `wget` call which renames original file with .1 suffix and writes new file to the intended filename. This should accomplish the intended behaviour